freeform_json_to_yaml.py

```diff
@@ -0,0 +1,392 @@
+                'heart_rate': PULSE_MAP[vitals['pulse']],
+                'spo2': 0 if vitals['SpO2'] == 'none' else 90 if vitals['SpO2'] == 'low' else 97
+            }
+            injuries = []
+            written_injuries = {}
+            for i in c['injuries']:
+                split_name = i['type'].split(' ')
+                severity = SEVERITY_MAP[i.get('bloodPool', 'None')]
+                if severity == 'minor':
+                    # no blood pool assigned, decide on our own
+                    if 'Puncture' in i['type']:
+                        severity = 'major'
+                    if 'Ampuation' in i['type']:
+                        severity = 'extreme'
+                    if 'Laceration' in i['type'] or 'Abrasion' in i['type']:
+                        severity = 'moderate' if 'Thigh' not in i['type'] else 'major'
+                    if 'Shrapnel' in i['type']:
+                        severity = 'major'
+                    if 'Chest Collapse' in i['type']:
+                        severity = 'extreme'
+                    if 'Burn' in i['type']:
+                        severity = 'major'
+                    if 'Broken Bone' in i['type']:
+                        severity = 'moderate'
+                injury = {
+                    'name': split_name[-1] if 'Broken' not in i['type'] else 'Broken Bone',
+                    'location': INJ_LOC_MAP[(' ').join(split_name[:-1])],
+                    'status': 'discoverable',
+                    'severity': severity
+                }
+                if injury['name'] in written_injuries:
+                    written_injuries[injury['name']].append(injury['location'])
+                else:
+                    written_injuries[injury['name']] = [injury['location']]
+                injuries.append(injury)
+            new_c['injuries'] = injuries
+            if len(written_injuries) == 0:
+                unstructured_inj = 'Has no known injuries.'
+            else:
+                unstructured_inj = "Has"
+                pronoun = 'his' if demographics['sex'] == 'M' else 'her'
+                for inj_set in written_injuries:
+                    if len(written_injuries[inj_set]) == 1:
+                        if inj_set == "Shrapnel":
+                            unstructured_inj += f" shrapnel in {pronoun} {written_injuries[inj_set][0]},".lower()
+                        elif inj_set == "Amputation":
+                            unstructured_inj += f" an amputated {written_injuries[inj_set][0]},".lower()
+                        else:
+                            unstructured_inj += f" a {inj_set} on {pronoun} {written_injuries[inj_set][0]},".lower()
+                            
+                    else:
+                        unstructured_inj += f" several {inj_set} injuries,".lower()
+                comma_separated = unstructured_inj.split(',')
+                unstructured_inj = ','.join(comma_separated[:-2]) + ' and' + comma_separated[-2] + '.'
```

Comment:
Not a request for change, but I am a supporter of the Oxford comma.

Reply:
Me too, I just didn't realize this removed it! But then I'd have to make sure if it's only two injuries we don't have it...